hw/xbox/nv2a/nv2a_vsh.c

```diff
@@ -423,16 +422,17 @@ static QString* decode_opcode(const uint32_t *shader_token,
             /* TODO : Emulate writeable const registers */
             qstring_append(ret, "c");
             qstring_append_int(ret,
-                convert_c_register(
-                    vsh_get_field(shader_token, FLD_OUT_ADDRESS)));
+                               convert_c_register(
+                                   vsh_get_field(shader_token,
+                                                 FLD_OUT_ADDRESS)));
         } else {
-            qstring_append(ret,
-                out_reg_name[
-                    vsh_get_field(shader_token, FLD_OUT_ADDRESS) & 0xF]);
+            qstring_append(
+                ret,
+                out_reg_name[vsh_get_field(shader_token, FLD_OUT_ADDRESS) &
+                             0xF]);
```

Comment:
I *strongly* prefer the old version. The new one manages to hide the mask or what it is applied to (or just makes it less easy to read).